example/src/index.css

```diff
@@ -16,6 +16,61 @@ label {
   margin-bottom: 32px;
 }
 
+/*CONTAINER - typograph.jsx*/
+.container > h1{
+  font-size: 35px;
+  line-height: 50px;
+}
+
+.container > h2{
+  font-size: 35px;
+  line-height: 48px;
+}
+
+.container > h3{
+  font-size: 30px;
+  line-height: 42px;
+}
+
+.container > h4{
+  font-size: 20px;
+  line-height: 30px;
+}
+
+.container > h5{
+  font-size: 14px;
+  letter-spacing: 0.8px;
+  line-height: normal;
+}
+
+.container > h6{
+  font-size: 11px;
+  line-height: normal;
+}
+
+.container > body{
+  font-size: 16px;
+  line-height: normal;
+}
+
+.container >  .underline{
+  text-decoration: underline;
+  font-size: 14px;
+  line-height: normal
+}
+
+.container > .small{
+  font-size: 14px;
+  line-height: 21px;
```

Comment:
Best met unitless line-heights werken + in variabelen stoppen.
https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0